packages/cozy-client/src/models/document/documentTypeData.js

```diff
@@ -230,5 +235,11 @@ export const themesList = [
     label: 'invoice',
     icon: 'bill',
     items: buildItemsByLabel(invoiceLabels)
+  },
+  {
+    id: 'theme10',
+    label: 'others',
+    icon: 'dots',
+    items: buildItemsByLabel(othersLabels)
```

Comment:
If we create an "other" theme, don't we want to include all the others labels? Like "other_work_document", etc?

Reply:
Good legitimate question, but not really, the idea is to place "unclassifiable" documents here and documents like "other_work_document" have an associated theme.